jobs/e2e_node/image-config.yaml

```diff
@@ -10,6 +10,6 @@ images:
     project: cos-cloud
     metadata: "user-data<test/e2e_node/jenkins/cos-init-live-restore.yaml,gci-update-strategy=update_disabled"
   cos-stable2:
-    image_regex: cos-stable-73-11647-510-0 # docker v18.09.7, deprecated after 2020-06-19
+    image_regex: cos-stable-81-12871-103-0 # docker v19.03.6
```

Comment:
nit: the convention I see here is that `stable1` should be the more recent image, can you swap this with above?

(we follow this same convention for kubernetes e2e jobs, `stable1` is newer than `stable2` is newer than `stable3` etc.)

Reply:
aggreed, was going to say the same.

Reply:
@sparkprime thank you for the review. updated as you've suggested.